src/core/InputGroup/InputGroup.tsx

```diff
@@ -75,7 +75,17 @@ export const InputGroup = (props: InputGroupProps) => {
   } = props;
   useTheme();
 
-  const icon = svgIcon ?? (status && StatusIconMap[status]());
+  const icon = () => {
```

Comment:
Why changed in InputGroup and LabeledSelect? What was wrong with the previous approach?